core/templates/domain/improvements/ExplorationTaskObjectFactory.ts

```diff
@@ -44,15 +44,22 @@ export type ExplorationTaskType = (
   'needs_guiding_responses' |
   'successive_incorrect_answers');
 
-export interface IExplorationTaskBackendDict extends ITaskEntryBackendDict {
-  'task_type': ExplorationTaskType;
-}
+export type IExplorationTaskBackendDict = (
```

Comment:
I thought we were dropping all the leading "I"s...

Also, adding @nishantwrp -- @nishantwrp, are you happy with the way these types are handled?

Reply:
Left a comment https://github.com/oppia/oppia/pull/9881#pullrequestreview-449876628

Reply:
I dropped the I for this type, but my changes predate the decision to drop Is so there are still some lying around. I can fix them up in a subsequent PR.